src/components/Table/index.js

```diff
@@ -23,34 +26,68 @@ export default class Table extends Component {
         }
     }
 
+    onColumnSelect(e, columnIndex) {
+        const { onSort } = this.props;
+        onSort(e, this.state.columns[columnIndex].field);
+        this.setState({ selectedColumn: columnIndex });
```

Comment:
this make that the table rerender completely and this is not needed, you can select the cell making it focusable